### ISSUE TITLE
Creating a group with a channel fails with "NotificationChannelGroup doesn't exist"

### ISSUE THREAD

**Issue by tad3j** [notifee/react-native-notifee#6]
Code
```
const groupId = await notifee.createChannelGroup({
    id: 'test_group',
    name: 'Test group',
  });

  const channelId = await notifee.createChannel({
    id: 'test_channel',
    name: 'Test Channel',
    groupId: 'test_group',
  });
```

Exception:

> Error: NotificationChannelGroup doesn't exist at Object.fn [as createChannel]
> at NotifeeApiModule.createChannel
> at onMessageReceived
> ...

Running Android emulator, Android9_Nexus_6_API_28...

Documentation says it should work: https://notifee.app/react-native/docs/android/channels#creating-updating-groups

**Comment by Salakar** [comment]
@tad3j we'll look into this, thanks for the report.

Are you able to provide the full stack trace / exception with line numbers etc?

**Comment by tad3j** [comment]
This is what I get in android debugger: 
![image](https://user-images.githubusercontent.com/4150904/73230176-a3ea1a80-417c-11ea-91db-b99a73cf42d2.png)

**Comment by Salakar** [comment]
Thanks, I think I know what this is now, its saying the channel group that was specified on the channel your creating doesn't exist, which is an issue in our code - will get it sorted tomorrow.

Error message could be better too so will look into that also.

**Comment by Ehesp** [comment]
Fixed this, will get a release up this morning.

**Comment by Salakar** [comment]
Released in 0.0.7. Thanks
